src/transformers/scss.js

```diff
@@ -7,12 +7,12 @@ module.exports = ({ content, filename, options }) => {
   options = {
     includePaths: getIncludePaths(filename),
     ...options,
+    data: options.data + content,
```

Comment:
if `options.data` is `undefined` this would prepend a `"undefined"` string to the file content, right?

Reply:
yes you're right, my bad 😱

could change this by `options.data ? options.data + content : content`

Reply:
That would work, yeah! It would also be great to move this logic to just below the `options = {}`, like on the [`sass-loader`](https://github.com/webpack-contrib/sass-loader/blob/master/lib/normalizeOptions.js#L37) for better readability 👍